Resources/Textures/Objects/Weapons/Melee/fireaxe.rsi/meta.json

```
@@ -0,0 +1,22 @@
+{
+    "version": 1,
+    "license": "CC-BY-SA-3.0",
```

Goon assets are all CC-BY-NC-SA-3.0.
```suggestion
    "license": "CC-BY-NC-SA-3.0",
```